bpf/tests/prog_test/prog_test.go

```diff
@@ -446,52 +451,52 @@ func testCt4Rst(spec *ebpf.Collection) error {
 	return nil
 }
 
-func main() {
+// TestCt checks connection tracking
+func TestCt(t *testing.T) {
 
-	lim := unix.Rlimit{
-		Cur: unix.RLIM_INFINITY,
-		Max: unix.RLIM_INFINITY,
-	}
-	err := unix.Setrlimit(unix.RLIMIT_MEMLOCK, &lim)
+	objDir := ".."
+	fname := path.Join(objDir, "bpf_ct_tests.o")
+	fnamePatched := path.Join(objDir, "bpf_ct_tests_patched.o")
+	err := patchElf(fname, fnamePatched)
```

Comment:
I see that this commit is changing a previous commit. Why not just squash these two commits together?

Reply:
squashed